Library/Homebrew/formula.rb

```diff
@@ -77,6 +77,9 @@ class Formula
   extend Cachable
   extend Attrable
   extend APIHashable
+  extend T::Helpers
+
+  abstract!
```

Comment:
This has a `raise` at line 225 when instantiating the base class, so it make sense to mark this `abstract!`. I left the check in place though, because direct instantiation fails for other, more cryptic, reasons before it gets to the sorbet check (which [requires](https://github.com/sorbet/sorbet/blob/c15a3f57173cc65c7144a5424cf6be682e6bca84/gems/sorbet-runtime/lib/types/private/abstract/declare.rb#L36-L42) the instantiation to complete successfully).